### PR TITLE
Add Sophy O as collaborator

### PR DESCRIPTION
**Pull request by ssssoz**
## Description

Just adding myself as collaborator after discussing with @lilyndk 

## Change Type
- [ ] Bug Fix
- [ ] New Feature
- [ ] Code Refactor
- [ ] Mentor Update
- [ ] Documentation
- [x] Other

### Before

screencapture-womencodingcommunity-github-io-team-2024-04-23-14_02_19

### After 

screencapture-127-0-0-1-4000-team-2024-04-23-14_02_08

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] I checked and followed the [contributor guide](CONTRIBUTING.md) 
- [x] I have tested my changes locally.
- [x] I have added a screenshot from the website after I tested it locally

### CODE REVIEW

File: _data/collaborators.yml [hunk 1]
```diff
@@ -35,3 +35,14 @@
       link: https://www.linkedin.com/in/nehha/
     - name: github
       link: https://github.com/Neha
+
+
+- name: Sophy O.
+  image: assets/images/collaborators/10.jpeg
+  position: UI Developer/UX Designer
+  comment: Team Collaborator
```
Comment (dricazenck) [inline _data/collaborators.yml]:
```suggestion
  comment: Website Collaborator
```

As a suggestion, you can add the topic you would be interested in collaborate, for example now it is the website.

But if you would like to collaborate with anything, please keep as `Team Collaborator`

File: _data/collaborators.yml [hunk 2]
```diff
@@ -35,3 +35,14 @@
       link: https://www.linkedin.com/in/nehha/
     - name: github
       link: https://github.com/Neha
+
```
Comment (lilyndk) [inline _data/collaborators.yml:38]: I would just remove the extra empty line here to keep consistent with the rest :)

Comment (dricazenck) [reply _data/collaborators.yml:38]: I guess is normal for some editor to add empty line. We can configure that on our editor and agree how do we want to do. I am ok, with both ways

**Review by dricazenck — APPROVED** [review]

File: _data/collaborators.yml (hunk 2 above)
Comment (ssssoz) [reply _data/collaborators.yml:38]: Done! And yes annoyingly my does it and so I stopped paying attention to that!

File: _data/collaborators.yml [hunk 3]
```diff
@@ -29,9 +29,20 @@
 - name: Neha Sharma
   image: assets/images/collaborators/5.jpeg
   position: Principal Engineer
-  comment: Team Collaborator
+  comment: Team Collaboratorg
```
Comment (lilyndk) [inline _data/collaborators.yml]: wrong change with the misspelled "Collaboratorg"?

Comment (dricazenck) [reply _data/collaborators.yml]:
```suggestion
  comment: Team Collaborator
```

**Pull request merged**